PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -974,6 +979,28 @@ local function CreateCamera()
 				OnKeyUp(input, processed)
 			end
 		end)
+		
+		MenuOpenedConn = GuiService.MenuOpened:connect(function()
+			isRightMouseDown = false
+			isMiddleMouseDown = false
+			OnMousePanButtonReleased() -- this function doesn't seem to actually need parameters
+			
+			if UserInputService.TouchEnabled then
+				--[[menu opening was causing serious touch issues
+				this should disable all active touch events if 
+				they're active when menu opens.]]
+				for inputObject, value in pairs(fingerTouches) do
+					fingerTouches[inputObject] = nil
+				end
+				panBeginLook = nil
```

Comment:
Should this be a standard reset function? Seems like we should just clear all input?